assembly/__tests__/u256.spec.ts

```diff
@@ -439,4 +504,56 @@ describe("Basic Operations", () => {
     expect(b * a).toStrictEqual(r);
   });
 
-});
+  it("Should divide two u256 numbers - divide by itself", () => {
+    const a = new u256(2, 0, 0, 0);
+    expect(a / a).toStrictEqual(u256.One);
+    expect(a % a).toStrictEqual(u256.Zero);
+  });
+
+  it("Should divide two u256 numbers - divide by 1", () => {
+    const a = new u256(2, 0, 0, 0);
+    expect(a / u256.One).toStrictEqual(a);
+    expect(a % u256.One).toStrictEqual(u256.Zero);
+  });
+
+  it("Should divide two u256 numbers - 0 divided", () => {
+    const a = new u256(2, 0, 0, 0);
+    expect(u256.Zero / a).toStrictEqual(u256.Zero);
+    expect(u256.Zero % a).toStrictEqual(u256.Zero);
+  });
+
+  /*it("Should divide two u256 numbers - divide by 0", () => {
+    expect(() => new u256(2, 0, 0, 0) / u256.Zero).toThrow();
+    expect(() => new u256(2, 0, 0, 0) % u256.Zero).toThrow();
+  });
+  */
```

Comment:
Why is this test commented ? 🤔

Reply:
Because it wasn't working. I found out by chance how to do it yesterday while doing another review. I will fix that.